applications/StructuralMechanicsApplication/custom_utilities/shell_utilities.h

```diff
@@ -20,52 +20,69 @@
 
 // Project includes
 #include "includes/define.h"
+#include "includes/properties.h"
 #include "containers/array_1d.h"
 
 namespace Kratos
 {
-	namespace ShellUtilities
-	{
-		template<class TVec>
-		inline void ShapeFunc(double Xi, double Eta, TVec& rN)
-		{
-			rN(0) = 0.25 * (1.0 - Xi) * (1.0 - Eta); // node 1
-			rN(1) = 0.25 * (1.0 + Xi) * (1.0 - Eta); // node 2
-			rN(2) = 0.25 * (1.0 + Xi) * (1.0 + Eta); // node 3
-			rN(3) = 0.25 * (1.0 - Xi) * (1.0 + Eta); // node 4
-		}
+namespace ShellUtilities
+{
+
+using SizeType = std::size_t;
+using IndexType = std::size_t;
+
+template<class TVec>
+inline void ShapeFunc(double Xi, double Eta, TVec& rN)
```

Comment:
well...this one is really in the geometry (LocalCoordinates). I am not asking to change it, just to take it into account for the future

Reply:
Thx for the hint @RiccardoRossi 
I will consider it in future iterations!